app/views/contribute.html

```diff
@@ -0,0 +1,186 @@
+{% extends 'layout.html' %}
+{% block pageTitle %}
+  Contribute - NHS prototype kit
+{% endblock %}
+{% block beforeContent %}
+  {{ breadcrumb({ href: "/", text: "Home" }) }}
+{% endblock %}
+{% block content %}
+
+{# the board is a long string to this is made a variable for consistency #}
+{% set board = "https://github.com/orgs/nhsuk/projects/24/views/1" %}
+{# the labels string is also tricky to making a variable #}
+{% set label = "?filterQuery=label%3A" %}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-two-thirds">
+    <h1 id="contribute-to-the-nhs-prototype-kit">
+      Contribute
+    </h1>
+    <p>
+      The NHS prototype kit is a community project supported by the NHS service
+      manual team. Anyone can help make it better.
+    </p>
+    <p>
+      <a href="{{board}}">View project board on GitHub</a>
+    </p>
+    <h2 id="why-contribute-to-the-kit">Why contribute to the kit</h2>
+    <p>If you contribute to the kit:</p>
+    <ul>
+      <li>
+        this keeps it up-to-date as an accessible and open way to make HTML
+        prototypes of NHS services
+      </li>
+      <li>
+        you will become part of an active and supportive community that will
+        help you learn and develop your skills with the kit
+      </li>
+      <li>
+        we will credit you in our update notes and other supporting information
+        unless you tell us not to (for example, because you want to stay
+        anonymous)
+      </li>
+    </ul>
+    <p>
+      If you are employed by NHS England, you will usually be able to use
+      examples of contributing to the kit as part of your yearly performance
+      review - talk to your line manager about how to include it in your
+      performance objectives.
+    </p>
+    <h2 id="how-you-can-contribute-to-the-kit">
+      How you can contribute to the kit
+    </h2>
+    <p>
+      There are many different ways that you can contribute, depending on your
+      skill and interests. You do not have to be an expert in coding.
+    </p>
+    <p>You can:</p>
+    <ul>
+      <li>add or improve content for people using the kit</li>
+      <li>
+        do quality assurance or accessibility testing of new features or
+        guidance
+      </li>
+      <li>fix bugs</li>
+      <li>add new features</li>
+      <li>improve the contribution process</li>
+    </ul>
+
+    <h3 id="add-or-improve-documentation-for-people-using-the-kit">
+      Add or improve content for people using the kit
+    </h3>
+    <p>You can also help improve the content for:</p>
+    <ul>
+      <li>this document site</li>
```

Comment:
"document site" feels weird? Guidance website? Or maybe even just "this website"?

```suggestion
      <li>this guidance website</li>
```